[IMP] accounting: turkiye localization (#19289)
odoo/documentation: content/applications/finance/fiscal_localizations/turkiye.rst

#. Open an existing delivery order or create a new one.
#. Once a :guilabel:`Delivery Address` is selected, fill in the necessary information in the
   :guilabel:`e-Dispatch` tab that appears.
#. Once the delivery order is validated, click the :guilabel:`Send GİB e-Dispatch (XML)` button that

>    appears to send the XML file. To only generate the file without sending it, click the

Reviewer: ```suggestion
   appears to send the XML file. To generate the file without sending it, click the
```

>    :icon:`fa-cog` :guilabel:`(gear)` icon, then click :guilabel:`Generate GIB e-Dispatch (XML)`.

Reviewer: ```suggestion
   :icon:`fa-cog` :guilabel:`(Actions)` icon, then click :guilabel:`Generate GIB e-Dispatch (XML)`.
```
Here we can use the tooltip as the icon descriptor

Document numbering structure
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

All generated invoices and dispatch documents must follow a specific numbering structure: `three
alphanumeric characters` + `document year` + `sequential number`.

.. example::
   INV/2025/00001

The numbering must be unique, sequential, and continuous. Each document must have a distinct
identifier.

Once a file has been submitted, even if it returns an error, the same sequence number cannot be
used; a new document must be created.

.. seealso::
   - :doc:`../accounting/customer_invoices/sequence`
   - :doc:`../accounting/vendor_bills/sequence`

e-Dispatch numbering structure
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The e-Dispatch numbering structure is accessed by navigating to :menuselection:`Inventory -->
> Configuration --> Operation Types`, and selecting an operation matching the :guilabel:`Delivery
> Orders` type.

Reviewer: ```suggestion
To access the e-Dispatch numbering structure, navigate to :menuselection:`Inventory -->
Configuration --> Operation Types`, then select an operation matching the :guilabel:`Delivery
Orders` type.
```
to use the active voice and be a bit more direct in the process

> You can set an arbitrary alphanumerical :guilabel:`Sequence Prefix` of your choice, but note that
> the system will only pick the **last three** characters of that sequence. The prefix must always
> contain at least three characters. Any prefix shorter than three characters is automatically denied
> by the system.

Reviewer: ```suggestion
An arbitrary alphanumerical :guilabel:`Sequence Prefix` can be set. The system uses only the
last three characters of that prefix. The prefix must contain at least three characters; any
shorter prefix is automatically denied by the system.
```

.. example::
   In the sequence WH/OUT/948224, the system will only send **224**.

.. tip::

>    You can preview the sequence from the :menuselection:`Inventory --> Configuration --> Operation
>    Types --> Delivery Orders` view. Once the sequencing is configured, the banner at the top shows a
>    preview:

Reviewer: ```suggestion
   You can preview the sequence by going to :menuselection:`Inventory --> Configuration --> Operation Types --> Delivery Orders`. Once the sequencing is configured, the banner at the top shows a preview:
```
or you can add the actual view's name but a menu is not a view

   .. image:: turkiye/banner-preview.png
      :alt: Banner preview of sequencing of e-Dispatch.

Sequential number gaps
~~~~~~~~~~~~~~~~~~~~~~

If a sequence number is skipped, the previous missing numbers must be issued within 7 days.

.. example::
   If INV/2025/00002 is issued before INV/2025/00001, INV/2025/00001 must be issued within 7 days.

No sequence number can remain unissued beyond this period.

Discount rules
~~~~~~~~~~~~~~

Global discounts (discounts shown as a separate invoice line) cannot be issued. All discounts must
be applied at the product line level within the invoice.

e-Ledger compliance
~~~~~~~~~~~~~~~~~~~

The first issued invoice must represent the opening balance for the accounting period. All

> subsequent invoices must follow the sequential numbering pattern:

Reviewer: ```suggestion
subsequent invoices must follow the sequential numbering pattern.
```

.. example::
   INV/2025/00001, INV/2025/00002, INV/2025/00003, etc.

Sequential numbering must be maintained consistently across the e-Ledger to ensure data integrity
and compliance.

Nilvera e-Invoice subscriptions
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. note::

>    Turkish e-Invoice law only permits one date range per invoice (at the header level), not per
>    individual line item. Because of this restriction, you cannot mix items with different
>    subscription date ranges on the same invoice. However, you can mix an item that has a
>    subscription date range with items that have no date range at all.
> 
> If the invoice is generated from the subscription order in the **Subscriptions** app, the system
> re-uses the date from the **Subscriptions** app as the invoice's date (at the header level).

Reviewer: ```suggestion
   Turkish e-Invoice law permits only one date range per invoice, at the header level, not per
individual line item. Therefore, items with different subscription date ranges cannot be mixed
on the same invoice. However, an item with a subscription date range can be mixed with items that have no date range at all.

If the invoice is generated from the subscription order in the **Subscriptions** app, the subscription order's date is used as the invoice's date, at the header level.
```
- passive voice here to avoid using you can/cannot (fits our style and guidelines better)
- I assume we mean the subscription order's date, if not, feel free to refine this further but imo "from the subsription app" is too vague
- and I'd try to fit in a doc link to the Subscription doc (or a ref to a relevant section) somwhere in this section

> If the invoice is manually generated, then the user has to set the subscription dates directly on
> the invoice lines.

Reviewer: ```suggestion
If the invoice is manually generated, then the subscription dates must be set directly on
the invoice lines.
```
we are addressing the user (with you/the imperative form) in the doc so switching to third person is confusing (however, here, I suggested to use the passive voice to avoid *you can/must xxx*

> If subscription dates do not match across lines, a warning is displayed.

Reviewer: ```suggestion
Subscription dates must match across lines.
```
to focus on what needs to be done instead of what happens if it's not done
